eng/pipelines/templates/steps/analyze.yml

```diff
@@ -27,7 +27,9 @@ steps:
   - pwsh: |
       $modDirs = ./eng/scripts/get_module_dirs.ps1 -serviceDir $(SCOPE)
       foreach ($md in $modDirs) {
-        Get-ChildItem $md/doc.go
+        if (-Not $md -Match "/arm") {
```

Comment:
Once my other change goes in we should probably switch this to using the type property from the metadata we read.